sdks/python/apache_beam/dataframe/io_test.py

```diff
@@ -410,5 +419,50 @@ def test_double_write(self):
                           set(self.read_all_lines(output + 'out2.csv*')))
 
 
+@unittest.skipIf(HttpError is None, 'GCP dependencies are not installed')
```

Comment:
What happens with read_gbq if GCP dependencies are not installed? Maybe we should not define it in that case?

Reply:
I tried without this, and the tests passed locally but failed on Jenkins and thus I wasn't able to easily reproduce the issue. Can discuss this offline if you think it's worth looking further into.

Reply:
Could you do an experiment to see what the error looks like if you try to use `read_gbq` without gcp dependencies installed? It should be sufficient to just make a new virtualenv, and install beam with `python -m pip install -e '.[test,dataframe]'` (note there's no gcp extra).

I'd like to make sure that the user gets a helpful error directing them to install GCP deps.

Even better would be to add a test that confirms this, it could be skipped unless GCP deps are _not_ installed.